yew/src/virtual_dom/key.rs

```diff
@@ -29,6 +30,15 @@ impl From<Rc<String>> for Key {
     }
 }
 
+impl Deref for Key {
+    type Target = str;
+
+    #[inline]
```

Comment:
What is the advantage of inlining this?

Reply:
Don't know what I was thinking, but at least it's also in `std::String`s impl marked as inline. Should I remove it?

Reply:
I guess explicitly inlining this might make LLVM do some more optimization work, but I don't think it really matters.